infrastructure-setup/scripts/prepare-sonarqube.sh

```diff
@@ -40,7 +40,8 @@ echo ${TOKEN_BASE64}
 cd ${BASE_DIR}
 find . -iname "sonarqube.env" -exec sed -i "s|AUTH_TOKEN=$sonar_admin_password_base64|AUTH_TOKEN=$TOKEN_BASE64|g" {} \;
 find . -iname "templates.env" -exec sed -i "s|SONAR_SERVER_AUTH_TOKEN=$sonar_admin_password_base64|SONAR_SERVER_AUTH_TOKEN=$TOKEN_BASE64|g" {} \;
-find . -iname "secret.env" -exec sed -i "s|SONAR_SERVER_AUTH_TOKEN=$sonar_admin_password_base64|SONAR_SERVER_AUTH_TOKEN=$TOKEN_BASE64|g" {} \;
+find . -iname "secret.env"    -exec sed -i "s|SONAR_SERVER_AUTH_TOKEN=$sonar_admin_password_base64|SONAR_SERVER_AUTH_TOKEN=$TOKEN_BASE64|g" {} \;
+find . -iname "ods-core.env"    -exec sed -i "s|SONAR_AUTH_TOKEN_B64=$sonar_admin_password_base64|SONAR_AUTH_TOKEN_B64=$TOKEN_BASE64|g" {} \;
```

Comment:
I guess all references here that are not to `ods-core.env` do not apply anymore, right?

Reply:
@michaelsauter thank you for your comment. I have implemented it.